src/components/Folder.tsx

```diff
@@ -66,7 +66,7 @@ export default function Folder({ source, config }: FolderProps) {
     searchElement?.addEventListener('keyup', handleKeyup)
     // Clean up event listener
     return () => searchElement?.removeEventListener('keyup', handleKeyup)
-  })
+  }, [filtered, source.prefix])
```

Comment:
hmmm, how did you detect that these dependencies were missing? Normally, eslint should have found them before, so maybe something is missing in the config?

Reply:
looks like exhaustive-deps doesn't apply if you have no dependencies at all?

Reply:
wow, weird. I'll look at it

Reply:
Indeed, it's the expected behavior, and there is a discussion to add a rule to detect when a hook is missing the dependencies array:

https://github.com/facebook/react/issues/22604